lib/kramdown/parser/gfm.rb

```diff
@@ -85,8 +85,8 @@ def parse_atx_header_gfm_quirk
         true
       end
 
-      FENCED_CODEBLOCK_START = /^[~`]{3,}/
-      FENCED_CODEBLOCK_MATCH = /^(([~`]){3,})\s*?((\S+?)(?:\?\S*)?)?\s*?\n(.*?)^\1\2*\s*?\n/m
+      FENCED_CODEBLOCK_START = /^\s*?[~`]{3,}/
```

Comment:
`\s*?` is not correct since after testing I found that only a maximum of three spaces is allowed by GFM on Github.